src/app/index-wrapper.jsx

```diff
@@ -12,22 +12,29 @@ function IndexWrapper() {
   const Keepalive = getAngularService('Keepalive');
   const authService = getAngularService('authService');
   const networkService = getAngularService('networkService');
+  const ngLocalStorage = getAngularService('$localStorage');
```

Comment:
Our convention has been to use the `$` to indicate AngularJS services that we use in our react stuff. My hope is that it will make it easier to find and remove them once they're no longer necessary

```suggestion
  const $localStorage = getAngularService('$localStorage');
```